src/api/tests/usersService.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { NextFunction, Response } from 'express';
+import { Knex } from 'knex';
+
+import { TestConfigure } from '../../database/TestSelfServeDatabase';
+import {
+  createParticipant,
+  createResponseObject,
+  createUser,
+} from '../../testHelpers/apiTestHelpers';
+import { enrichWithUserFromParams, UserRequest } from '../services/usersService';
+
+const createUserRequest = (email: string, userId: string | number): UserRequest => {
+  return {
+    auth: {
+      payload: {
+        email,
+      },
+    },
+    params: {
+      userId,
+    },
+  } as unknown as UserRequest;
```

Comment:
I don't love this but I couldn't figure out a better way to do this. Open to ideas

Reply:
Again, this makes sense. This is actually good - what we're doing isn't "correct" (in that this *isn't* a UserRequest), but it's close enough for this test. So we tell TypeScript "shhhhh.... this is wrong but it's OK for the purpose of this test".